script.js

```diff
@@ -19,4 +20,310 @@ closeMenuBtn.addEventListener('click', menuToggler);
+    mobliveLink: '',
+    mobsourceLink: '',
+    mobimgPos: 'img-center',
+    mobSubHead: 'card-sub-head',
+    cardMob: 'card',
+    mobcanopy: 'canopy',
+    mobsubText: 'sub-text',
+    mobfaCircle: 'fa-circle',
+    mobAlt: 'Project 2',
+  },
+
+  {
+    mobprojName: 'Tonic',
+    mobprojDetails: ['CANOPY', 'Back End Dev', '2015'],
+    mobshortDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    moblongDesc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    mobtonicImgUrl: 'mobile-images/works/Portfolio3.png',
+    mobtechlgsUsed: ['html', 'css', 'javascript'],
+    mobPopUpImgUrl: 'mobile-images/works/Portfolio3.png',
+    mobtechlgsPopup: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+    mobliveLink: '',
+    mobsourceLink: '',
+    mobimgPos: 'img-center',
+    mobSubHead: 'card-sub-head',
+    cardMob: 'card',
+    mobcanopy: 'canopy',
+    mobsubText: 'sub-text',
+    mobfaCircle: 'fa-circle',
+    mobAlt: 'Project 3',
+  },
+
+  {
+    mobprojName: 'Multi-Post Stories',
+    mobprojDetails: ['CANOPY', 'Back End Dev', '2015'],
+    mobshortDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    moblongDesc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    mobtonicImgUrl: 'mobile-images/works/Portfolio2.png',
+    mobtechlgsUsed: ['html', 'css', 'javascript'],
+    mobPopUpImgUrl: 'mobile-images/works/Portfolio2.png',
+    mobliveLink: '',
+    mobsourceLink: 'https://github.com/mwenyoa/Portfolio',
+    mobimgPos: 'img-center',
+    mobSubHead: 'card-sub-head',
+    cardMob: 'card',
+    mobcanopy: 'canopy',
+    mobsubText: 'sub-text',
+    mobfaCircle: 'fa-circle',
+    mobAlt: 'Project 4',
+  },
```

Comment:
[optional] it will be better to use one array of objects for both your desktop and mobile section since they have similar contents, This will improves maintenance time and bug when you try to update a project